pkg/cmd/server/apis/config/v1/types.go

```diff
@@ -523,7 +523,7 @@ type UserAgentMatchRule struct {
 
 // UserAgentDenyRule adds a rejection message that can be used to help a user figure out how to get an approved client
 type UserAgentDenyRule struct {
-	UserAgentMatchRule `json:", inline"`
+	UserAgentMatchRule `json:"inline"`
```

Comment:
This just broke serialization.  How did it pass tests?  `json:",inline"`, right?